com.unity.ml-agents/Runtime/Communicator/UnityRLCapabilities.cs

```diff
@@ -5,6 +5,7 @@ namespace Unity.MLAgents
     internal class UnityRLCapabilities
     {
         internal bool m_BaseRLCapabilities;
+        internal bool m_ConcatenatedPngObservations;
```

Comment:
I am not sure that internal variables need the `m_` prefix.

Reply:
I'm not sure either. I can just make them public since it's an internal class.

Reply:
I think it is better yes.